### PR TITLE
Fix some more defects and some minor tidying

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: plugins/ftdidmx/FtdiDmxPlugin.cpp [hunk 1]
```diff
@@ -139,8 +138,9 @@ bool FtdiDmxPlugin::SetDefaultPreferences() {
 int unsigned FtdiDmxPlugin::GetFrequency() {
   unsigned int frequency;
 
-  if (!StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency))
-    StringToInt(DEFAULT_FREQUENCY, &frequency);
+  if (!StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency)) {
```
Comment (nomis52) [inline plugins/ftdidmx/FtdiDmxPlugin.cpp]: I'd invert this, I think it's easier to read :+1: 

if (StringToInt(m_preferences->GetValue(K_FREQUENCY), &frequency)) {
  return frequency;
}
return DEFAULT_FREQUENCY;

File: plugins/usbpro/UsbSerialPlugin.cpp [hunk 2]
```diff
@@ -361,8 +359,9 @@ string UsbSerialPlugin::GetDeviceName(
 unsigned int UsbSerialPlugin::GetProFrameLimit() {
   unsigned int fps_limit;
   if (!StringToInt(m_preferences->GetValue(USB_PRO_FPS_LIMIT_KEY) ,
-                   &fps_limit))
-    StringToInt(DEFAULT_PRO_FPS_LIMIT, &fps_limit);
+                   &fps_limit)) {
```
Comment (nomis52) [inline plugins/usbpro/UsbSerialPlugin.cpp]: Same here

File: plugins/ftdidmx/FtdiDmxPlugin.cpp (hunk 1 above)
Comment (peternewman) [inline plugins/ftdidmx/FtdiDmxPlugin.cpp]: I'm actually thinking I might create a StringToIntOrDefault which wraps it all in one function.

Comment (peternewman) [inline plugins/ftdidmx/FtdiDmxPlugin.cpp]: Switched.

File: plugins/usbpro/UsbSerialPlugin.cpp (hunk 2 above)
Comment (peternewman) [inline plugins/usbpro/UsbSerialPlugin.cpp]: Switched.

**Pull request merged**